layers/+source-control/git/packages.el

```diff
@@ -154,6 +155,11 @@ Press [_b_] again to blame further in the history, [_q_] to go up or quit."
       (require 'git-rebase)
       ;; bind function keys
       ;; (define-key magit-mode-map (kbd "<tab>") 'magit-section-toggle)
+      (evilified-state-evilify-map magit-repolist-mode-map
+          :mode magit-repolist-mode
+          :bindings
+          (kbd "g") 'magit-list-repositories
```

Comment:
I think it should be bound to `gr` as we do in other buffers to _refresh_ the view. Binding it to `g` would override some evil bindings such `gg` to goto the top of buffer.